source/options.tsx

```diff
@@ -62,6 +62,17 @@ async function init(): Promise<void> {
 	fitTextarea.watch('textarea');
 	indentTextarea.watch('textarea');
 
+	// Search feature options
+	select<HTMLInputElement>('#search-features')!.addEventListener('input', event => {
+		const keywords = (event.target as HTMLInputElement)!.value.toLowerCase().split(' ').filter((s: string) => s.length);
+		const hasKeywords = keywords.length > 0;
+		for (const feature of select.all('.feature')) {
+			feature.hidden = hasKeywords ?
+				!keywords.every(word => feature.dataset.text.includes(word)) :
+				false;
```

Comment:
This shouldn't be necessary. I _think_ this is equal:

```js
feature.hidden = !keywords.every(word => feature.dataset.text.includes(word));
```

Reply:
You're right... like I said my computer was doing some crazy stuff. I cleared the input and all the features stayed hidden. I thought I was going crazy. That and the times the input would enter text in backwards.